tests/ccs.nf.test

```diff
@@ -0,0 +1,56 @@
+nextflow_pipeline {
+
+    name "Test pipeline"
+    script "../main.nf"
+    tag "pipeline"
+    tag "pipeline_isoseq"
+
+    test("-profile test_ccs") {
+
+        when {
+            params {
+                outdir          = "$outputDir"
+                input           = "$projectDir" + '/assets/samplesheet_ccs.csv'
+                primers         = 'https://raw.githubusercontent.com/nf-core/test-datasets/isoseq/testdata/primers.fasta'
+                fasta           = 'https://raw.githubusercontent.com/nf-core/test-datasets/isoseq/reference/Homo_sapiens.GRCh38.dna.chromosome.19.fasta'
+                gtf             = 'https://raw.githubusercontent.com/nf-core/test-datasets/isoseq/reference/Homo_sapiens.GRCh38.104.chr.13_18_19.gtf'
```

Comment:
you should try using the pipeline test data base path params there

Reply:
@maxulysse I updated the test to use the `params.pipelines_testdata_base_path` from `tests/nextflow.config`. 

```
primers = params.pipelines_testdata_base_path + 'testdata/primers.fasta'
                fasta   = params.pipelines_testdata_base_path + 'reference/Homo_sapiens.GRCh38.dna.chromosome.19.fasta'
                gtf     = params.pipelines_testdata_base_path + 'reference/Homo_sapiens.GRCh38.104.chr.13_18_19.gtf'
```

But, when nf-test is run, the variable is not resolved correctly and gives `null` instead of `https://raw.githubusercontent.com/nf-core/test-datasets/refs/heads/isoseq/`. 

I can't figure out what is going wrong. 
Do you have an idea of what happening?

Thanks in advance for your insights.